### ISSUE TITLE
Re-spawning projection behavior does not resume projection

### ISSUE THREAD

**Issue by claudio-scandura** [akka/akka-projection#423]
### Versions used 

akka-projection-core:1.0.0-RC3

Akka version: 2.6.8

### Expected Behavior

When running a projection with the `ProjectionBehaviour`, after sending a `ProjectionBehavior.Stop` command and subsequently re-spawning the actor, the projection should resume from the last committed offset. 

### Actual Behavior
The last committed offset is read but the projection does not resume.

### Reproducible Test Case
Added a test that reproduces the issue [here](https://github.com/akka/akka-projection/compare/master...claudio-scandura:restart-projection?expand=1)

I am not entirely sure this is a bug, a misuse of the API, or my misinterpretation of the API. On a more realistic example I have with a Kafka-to-Postgres projection I have pinpointed (with the debugger) the issue to be within the `mappedSource()` code in the `InternalProjectionState`. Looks like when the projection is restarted there are references to the old InternalProjectionState object going around.

**Comment by patriknw** [comment]
@claudio-scandura Can you show how you tried this? The link to "test that reproduces the issue" is invalid.

**Comment by renatocaval** [comment]
@patriknw, the link @claudio-scandura published here was to a github compare to create the PR. 

The PR is now create (see #424). Let's see what happens in Travis build.

**Comment by claudio-scandura** [comment]
Yes, sorry I had deliberately not created the PR as I wanted to ensure the use case is valid first.
